pkg/shell/hosts_dialog.jsx

```diff
@@ -475,7 +478,24 @@ class HostKey extends React.Component {
                 <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-fingerprint pf-v5-u-font-family-monospace">{fp}</ClipboardCopy>
                 <p className="hostkey-type">({key_type})</p>
                 <p>{cockpit.format(_("To verify a fingerprint, run the following on $0 while physically sitting at the machine or through a trusted network:"), this.props.host)}</p>
-                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">ssh-keyscan -t {key_type} localhost | ssh-keygen -lf -</ClipboardCopy>
+                <ClipboardCopy isReadOnly hoverTip={_("Copy")} clickTip={_("Copied")} className="hostkey-verify-help-cmds pf-v5-u-font-family-monospace">{`ssh-keyscan -t ${key_type} localhost | ssh-keygen -lf -`}</ClipboardCopy>
+                <p>{_("Optionally, paste the result below to compare fingerprint")}</p>
+                <Form>
+                    <FormGroup>
+                        <TextArea id="command-ssh-keyscan-result-paste" spellCheck="false" placeholder="# localhost:22 SSH-..."
```

Comment:
I don't think we need a placeholder.

Reply:
I gave a thought about this, but removed it anyways for now.

Should the placeholder be included to suggest the user that "localhost:22" will be present when the command is run locally, supporting "_Run this command over a trusted network or physically on the remote machine_"?